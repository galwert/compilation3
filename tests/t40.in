int func(int param1, int arg2) {
    return 1;
}

void main(){
    printi(func(1, 2)); // Should be fine
}