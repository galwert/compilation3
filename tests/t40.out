---end scope---
param1 INT -1
arg2 INT -2
---end scope---
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
func (INT,INT)->INT 0
main ()->VOID 0
